### PR TITLE
monochrome

### PR DESCRIPTION
**Pull request by soundsnow**
Screenshot_20240612-112539_Trebuchet

### CODE REVIEW

**Comment by torokati44** [comment]
Could you please explain briefly what exactly this change achieves?
The screenshot looks irrelevant, I'm not seeing the Ruffle application there.

File: app/build.gradle.kts [hunk 1]
```diff
@@ -18,7 +18,7 @@ android {
 
     defaultConfig {
         applicationId = "rs.ruffle"
-        minSdk = 26
+        minSdk = 14
```
Comment (torokati44) [inline app/build.gradle.kts:21]:
```
 /home/runner/work/ruffle-android/ruffle-android/app/src/main/AndroidManifest.xml Error:
	uses-sdk:minSdkVersion 14 cannot be smaller than version 21 declared in library [androidx.appcompat:appcompat-resources:1.7.0] /home/runner/.gradle/caches/transforms-4/6e509871dbb0a44646463976bf102953/transformed/appcompat-resources-1.7.0/AndroidManifest.xml as the library might be using APIs not available in 14
	Suggestion: use a compatible library with a minSdk of at most 14,
		or increase this project's minSdk version to at least 21,
		or use tools:overrideLibrary="androidx.appcompat.resources" to force usage (may lead to runtime failures)
> Task :app:processReleaseMainManifest FAILED
```

**Review by torokati44 — CHANGES_REQUESTED** [review]

**Comment by torokati44** [comment]
This initially tiny PR has exploded in scope - and note that there is no point in having dozens of layouts that don't actually do anything.

**Comment by torokati44** [comment]
Before you continue further, please define the rough scope and goal of _this specific PR_, and keep the changes in adherence to it. Without this, I'll have to close this PR.

**Comment by adrian17** [comment]
Sorry, the part you're missing is that we aren't interested in anything that has to do with Magic DosBox, and especially their (GPL) source.

**Pull request closed without merging**